hack/update-codegen-dockerized.sh

```diff
@@ -58,6 +59,7 @@ $GOPATH/bin/deepcopy-gen \
   --input-dirs "${ANTREA_PKG}/pkg/apis/system/v1beta1" \
   --input-dirs "${ANTREA_PKG}/pkg/apis/security/v1alpha1" \
   --input-dirs "${ANTREA_PKG}/pkg/apis/core/v1alpha1" \
+  --input-dirs "${ANTREA_PKG}/pkg/apis/traceflow/v1alpha1" \
```

Comment:
```suggestion
  --input-dirs "${ANTREA_PKG}/pkg/apis/ops/v1alpha1" \
```
This should be the group name